app/javascript/packages/document-capture/components/in-person-location-full-address-entry-post-office-search-step.spec.tsx

```diff
@@ -74,7 +76,10 @@ describe('InPersonLocationFullAddressEntryPostOfficeSearchStep', () => {
   beforeEach(() => {
     server.resetHandlers();
     // todo: should we return USPS_RESPONSE here?
-    server.use(rest.post(locationsURL, (_req, res, ctx) => res(ctx.json([{ name: 'Baltimore' }]))));
+    server.use(
+      rest.post(locationsURL, (_req, res, ctx) => res(ctx.json([{ name: 'Baltimore' }]))),
+      rest.put(locationsURL, (_req, res, ctx) => res(ctx.json({ success: true }))),
```

Comment:
was this like a missing stub?

Reply:
> was this like a missing stub?

Yes, with [the previous code](https://github.com/18F/identity-idp/blob/cf851c83be6672cff0846d5112d76c5b8af5899a/app/javascript/packages/document-capture/components/in-person-location-full-address-entry-post-office-search-step.spec.tsx#L329), we weren't awaiting `findAllByText`, so the thing it was "clicking" was a promise object, not the intended button element. Clicking the button [issues a network request before redirecting](https://github.com/18F/identity-idp/blob/cf851c83be6672cff0846d5112d76c5b8af5899a/app/javascript/packages/document-capture/components/in-person-location-full-address-entry-post-office-search-step.tsx#L61-L79). Once the correction was made to await the result to click the button, these changes were also necessary to stub the resulting network response.